mifospay/src/main/res/layout/activity_login.xml

```diff
@@ -12,31 +12,17 @@
 
     <ImageView
         android:id="@+id/image_logo"
-        android:layout_width="300dp"
-        android:layout_height="160dp"
-        android:src="@drawable/mifos_icon"
-        app:layout_constraintBottom_toTopOf="@+id/textView3"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-
-    <TextView
-        android:id="@+id/textView3"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginVertical="@dimen/value_10dp"
-        android:layout_marginTop="22dp"
-        android:layout_marginBottom="26dp"
-        android:text="@string/mifos_pay"
-        android:textColor="@color/colorAccent"
-        android:textSize="@dimen/textH4"
+        android:layout_height="150dp"
+        android:layout_marginBottom="50dp"
```

Comment:
hardcoded dimension

Reply:
Well, I see there are some files in the project which have hardcoded dimensions. Do you want me to make a pr on this issue?